src/i18n/hi/ui.ts

```diff
@@ -54,5 +54,81 @@ export default UIDictionary({
 	'cms.navTitle': 'और CMS मार्गदर्शिकाएँ', // "More CMS guides"
 	// Migration Guides vocabulary
 	'migration.navTitle': 'और माइग्रेशन मार्गदर्शिकाएँ', // "More migration guides"
-	// Recipes
+	// Recipes vocabulary
+	'recipes.navTitle': 'और अधिक रेसिपीस', // "More recipes"
+	// `<RecipeLinks>` vocabulary
+	'recipesLink.singular': 'संबंधित रेसिपी:', // "Related recipe:"
+	'recipesLink.plural': 'संबंधित रेसिपीस', // "Related recipes"
+	// `<ContributorList>` fallback text
+	'contributors.seeAll': 'सभी योगदानकर्ताओं को देखें', // "See all contributors"
+	// Fallback content notice shown when a page is not yet translated
+	'fallbackContent.notice':
+		'यह पेज अभी तक आपकी भाषा में उपलब्ध नहीं है, इसलिए हम आपको अंग्रेजी संस्करण दिखा रहे हैं। आप इसका अनुवाद करके मदद कर सकते हैं!', // "This page is not yet available in your language, so we’re showing you the English version. You can help by translating it!"
+	'fallbackContent.linkText': 'आप कैसे योगदान दे सकते हैं, इसके बारे में और जानें', // "Learn more about how you can contribute"
+	// 404 Page
+	'404.title': 'नहीं मिला', // "Not Found"
+	'404.content': 'यह पेज हमारे सौरमंडल में नहीं है।', // "This page isn’t in our solar system."
+	'404.linkText': 'मुझे घर ले चलो।', // "Take me home."
+	// Aside component default labels
+	'aside.note': 'टिप्पणी', // "Note"
+	'aside.tip': 'सुझाव', // "Tip"
+	'aside.caution': 'सावधानी', // "Caution"
+	'aside.danger': 'खतरा', // "Danger"
+	// `<LanguageSelect>` vocabulary
+	'languageSelect.label': 'भाषा चुने', // "Select language"
+	// Integrations vocabulary
+	'integrations.changelog': 'चेंजलॉग', // "Changelog"
```

Comment:
It should be 'परिवर्तन लॉग' or you can keep it in English as is.

Reply:
solid catch 💯 applying it